### PR TITLE
Place 'please remember to log in' message on top of mashup page #2854

### LINKED ISSUE #2854: Place 'please remember to log in' message on top of mashup page
mashup.jsp page doesn't work if the user is not logged in (as admin?)

### PR DESCRIPTION
**Pull request by supunharsha**
Fixes #2854

### CODE REVIEW

File: .settings/com.google.appengine.eclipse.core.prefs [hunk 1]
```diff
@@ -3,6 +3,7 @@ filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.4.jar|appengine-api-labs.jar|ap
 gaeDatanucleusVersion=v1
 gaeDeployDialogSettings=
 gaeIsEclipseDefaultInstPath=true
+gaeLaunchAppInBrowser=true
```
Comment (kanghj) [inline .settings/com.google.appengine.eclipse.core.prefs]: Remove files that are not related to the issue.

File: src/main/webapp/dev/mashup.jsp [hunk 2]
```diff
@@ -2,354 +2,601 @@
 <%@ page import="teammates.common.util.StringHelper"%>
 <%@ page import="teammates.logic.core.FeedbackQuestionsLogic"%>
 <%@ page import="teammates.logic.api.Logic"%>
-<%@ page import="teammates.common.datatransfer.StudentAttributes" %>
+<%@ page import="teammates.common.datatransfer.StudentAttributes"%>
```
Comment (kanghj) [inline src/main/webapp/dev/mashup.jsp]: Remove unrelated changes due to indentation from the changeset. The pull request should only contain the required changes for the issue to be completed.

File: src/test/resources/pages/godmode.html [hunk 3]
```diff
@@ -15,8 +15,8 @@
             type="text"
             name="start"
             id="start"
-            value="<!-- DATETODAY -->">
-        <p class="form-control-static"><!-- DATETIMETODAY --></p>
+            value="08/03/2015">
```
Comment (kanghj) [inline src/test/resources/pages/godmode.html]: This file should be removed from the changes as well

File: src/main/webapp/dev/mashup.jsp [hunk 4]
```diff
@@ -2,354 +2,601 @@
...
+
+<link href="/bootstrap/css/bootstrap.min.css" rel="stylesheet"
+    type="text/css" />
+<link href="/bootstrap/css/bootstrap-theme.min.css" rel="stylesheet"
+    type="text/css" />
+<link href="/stylesheets/teammatesCommon.css" rel="stylesheet"
+    type="text/css" />
+<link rel="stylesheet" type="text/css" href="./mashup.css">
+<style type="text/css">
+#apDiv1 {
```
Comment (kanghj) [inline src/main/webapp/dev/mashup.jsp]: There seems to be some css you added here which are not used at all

**Comment by kanghj** [comment]
Can you also post a screenshot of the message you've added?

**Comment by LowWeiLin** [comment]
Title for the pull request is in the wrong format.

**Comment by supunharsha** [comment]
changes in mashup.jsp

login

**Comment by rcyq** [comment]
image


You can try to put the message in a warning panel as showed above.

**Comment by supunharsha** [comment]
Hi

I have added those css properties as requested. You can let me know if
colors needs to be changed.

[image: Inline image 1]

**Comment by kanghj** [comment]
Hi @supunharsha, can you post a screenshot again? Your previous screenshot did not seem to have uploaded before you submitted the comment. In future, try to name your commits more meaningfully.

**Comment by supunharsha** [comment]
mashup

**Comment by kanghj** [comment]
@supunharsha, the css you added does not seem consistent with the rest of the UI that we are using now. Can you change them to follow what @rcyq suggested above? You should be able to find examples of it from various pages on the dev server, or from the bootstrap website.

**Comment by supunharsha** [comment]
Hi

I have done as you asked. 
login

If you don't mind can you explain me how my names for commits are wrong. I have been using the same because it should be related to the issue. Please correct me if i am wrong. I highly appreciate that.

**Comment by kanghj** [comment]
Every commit message is the same now and we will not be able to distinguish any 2 commits from each other. The commit message should instead mention what you changed in the commit. For example, in your last commit, you modified the class of the message, so your commit message should mention that.


This looks good to me.

**Comment by supunharsha** [comment]
Thanks for the reply.

How can i proceed now?May i ask for another issue?

**Comment by kanghj** [comment]
@supunharsha, yes, you may look for another issue to work on.
